webcompat/helpers.py

```diff
@@ -27,6 +31,37 @@ def format_delta_filter(timestamp):
     return format_timedelta(delta, locale='en_US')
 
 
+@app.template_filter('bust_cache')
+def bust_cache(file_path):
+    '''Jinja2 filter to add a cache busting param based on md5 checksum.
+
+    Uses a simple cache_dict to we don't have to hash each file for every
+    request. This is kept in-memory so it will be blown away when the app
+    is restarted (which is when file changes would have been deployed).'''
+    def get_checksum(file_path):
+        try:
+            checksum = cache_dict[file_path]
+        except KeyError:
+            checksum = md5_checksum(file_path)
+            cache_dict[file_path] = checksum
+        return checksum
+
+    return file_path + '?' + get_checksum(STATIC_PATH + file_path)
+
+
+def md5_checksum(file_path):
+    '''Return the md5 checksum for a given file path.'''
+    with open(file_path, 'rb') as fh:
+        m = hashlib.md5()
+        while True:
+            # only read in 8k of the file at a time
+            data = fh.read(8192)
+            if not data:
+                break
+            m.update(data)
+        return m.hexdigest()
```

Comment:
just for my own understanding and to be sure it was doing the right thing.

``` python
>>> import hashlib
>>> m = hashlib.md5()
>>> m
<md5 HASH object @ 0x107a0c230>
>>> m.hexdigest()
'd41d8cd98f00b204e9800998ecf8427e'
>>> fullstring = 'ABC'
>>> n = hashlib.md5()
>>> n.hexdigest()
'd41d8cd98f00b204e9800998ecf8427e'
>>> n.update(fullstring)
>>> n.hexdigest()
'902fbdd2b1df0c4f70b4a5d23525e932'
>>> m.update('A')
>>> m.update('B')
>>> m.update('C')
>>> m.hexdigest()
'902fbdd2b1df0c4f70b4a5d23525e932'
>>> m.hexdigest() == n.hexdigest()
True
```

Good to go!